internal/graph/check.go

```diff
@@ -175,6 +179,8 @@ type LocalChecker struct {
 	delegate           CheckResolver
 	concurrencyLimit   uint32
 	maxConcurrentReads uint32
+	usersetBatchSize   uint32
+	logger             logger.Logger
```

Comment:
This is not being used 🤔

Reply:
Oh right I used this add debug logs but then I removed them. I still think it's a good idea to have the logger available though..

Reply:
If not going to be used, remove it.

Reply:
But then next time i'm debugging something in Check I will need it again, i'd rather leave it here, what is the cost?